pkgs/applications/audio/jack-autoconnect/default.nix

```diff
@@ -0,0 +1,38 @@
+# Call with libsForQt5.callPackage
```

Comment:
I don't think this comment is necessary. (If you disagree, do you mind explaining why? Happy to hear your opinion on this.)

Reply:
@winterqt I don’t mind removing it since many modules that depend on Qt do not include this comment anyway. But this is an implicit knowledge that if you have Qt dependencies you need to call it using `libsForQtN.callPackage`. Also it might be not obvious that it depends on Qt5, not Qt4 for instance. I’ll remove the comment anyway. Let’s say we have a usage example in `all-packages.nix`.

Reply:
Removed the comment.